extensions.json

```
@@ -270,6 +270,11 @@
       "version": "5.1.3",
       "checkout": "v5.1.3"
     },
+    {
+      "id": "ethan-reesor.vscode-go-test-adapter",
+      "repository": "https://gitlab.com/firelizzard/vscode-go-test-adapter",
+      "version": "0.1.3"
```

Since we're publishing from the repository's default branch (i.e. there are no dedicated release branches or tags to `"checkout"`), I think it's not necessary to pin the `"version"`.

```suggestion
      "repository": "https://gitlab.com/firelizzard/vscode-go-test-adapter"
```